Applications/SimpleShot/simpleshot_utils.cpp

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file   utils.cpp
+ * @date   08 Jan 2021
+ * @brief  This file contains simple utilities used across the application
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug    No known bugs except for NYI items
+ *
+ */
+#include <iostream>
+#include <regex>
+
+#include <simpleshot_utils.h>
+
+namespace simpleshot {
+
+namespace util {
+
+Entry getKeyValue(const std::string &input) {
+  Entry entry;
```

Comment:
Which one is better? make this kind of API or leave it for the user?

Reply:
I'd go for API, I will make this as an api with property functionality :)